style.css

```diff
@@ -463,23 +422,289 @@ header {
 }
 
 .submit-contact {
-  width: 131px;
-  margin-left: 98px;
-  margin-top: -12px;
+  width: 40.06%;
 }
 
 footer {
-  padding: 25px 95px;
+  width: 100%;
+  padding: 25px 0 5px 0;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
   border-top: 1px solid #dfe1e6;
   margin: 124px 0 8px 0;
+  box-sizing: border-box;
 }
 
 .footerIcon {
-  filter: invert(80%);
+  filter: invert(70%);
 }
 
-.headerLine {
+.footerLine {
   border-top: 5px solid #0d151e;
-  margin-left: 36px;
   width: 134px;
 }
+
+@media (min-width: 768px) {
+  /* header block */
+
+  .headTabs {
+    display: grid;
+    grid-template-columns: auto auto auto auto;
+    column-gap: 32px;
+    justify-content: right;
+  }
+
+  .unionImg {
+    display: none;
+  }
+
+  header {
+    width: 100%;
+    padding-left: 7%;
+  }
+
+  main {
+    width: 100%;
+  }
+
+  #headline {
+    height: 870px;
+    background-image: url("Icons/headline-big-background.svg");
+    background-repeat: no-repeat;
+    background-position-x: 100%;
+    background-position-y: 0;
+    display: flex;
+    flex-direction: row-reverse;
+    align-items: center;
+    justify-content: left;
+  }
+
+  #title {
+    margin: 0 0 12px 0;
+    width: 30%;
+    padding-right: 12px;
+    font-size: 3.5em;
+    line-height: 62px;
+  }
+
+  #subtitle {
+    font-size: 2em;
+    line-height: 44px;
+  }
+
+  #mainDescription {
+    width: 40%;
+    padding-top: 12px;
+  }
+
+  .socialMedia {
+    display: flex;
+    flex-direction: column;
+    margin-left: 1%;
+    margin-right: 2%;
+  }
+
+  .imgSoc1 img {
+    width: 1.65em;
+    height: 1.65em;
+  }
+
+  /* work block */
+
+  #workTitle-label {
+    display: grid;
+    grid-template-columns: auto auto;
+    align-items: top;
+    column-gap: 1px;
+    margin-top: 135px;
+    justify-content: center;
+  }
+
+  #workTitle {
+    border: none;
+    height: 1px;
+    padding: 0;
+    margin: 0;
+    font-size: 2.5em;
+  }
+
+  .empty1 {
+    border-top: 1px solid black;
+    height: 1px;
+    width: 55em;
+    margin-left: 24px;
+  }
+
+  #storyBox {
+    display: flex;
+    justify-content: center;
+    margin-left: 25%;
+  }
```

Comment:
- [ ] [Optional] Kindly make sure that your design matches that of the Figma template
![Capture1](https://user-images.githubusercontent.com/85095702/165848740-4f6d6daa-b83c-4fe3-a042-3f3423014836.PNG)

Reply:
@[phelian23](https://github.com/phelian23) It is like in the Figma template, but when you decrease the screen, it changes, because if I wasn't using percentage for this width it couldn't fit when the screen decreases.